tests/swagger_generation/test_generator_utils.py

```diff
@@ -217,49 +217,3 @@ def test_no_args(self):
 
         self.assertEqual(res, "/health")
         self.assertEqual(args, tuple())
-
-
-class MyCustomAuthenticator(Authenticator):
-    def authenticate(self):
-        pass
-
-
-def convert_my_customer_authenticator(authenticator):
-    definition = {"type": "basic"}
-    return "myAuth", definition
-
-
-class TestAuthenticatorConverter(unittest.TestCase):
-    def test_legacy_behaviour(self):
```

Comment:
I think this was the only AuthenticatorConverter test we may have had (My bad there).
Should we replace this with a test that check the new interface instead of just removing it?

Reply:
I looked around and thought I saw a couple of tests for the new approach but it's entirely possible I just glanced sideways at it ;) -- will take a better look and def agree if it's missing will add some tests